assets/src/block-editor/helpers/index.js

```diff
@@ -274,15 +274,17 @@ export const filterBlocksEdit = ( BlockEdit ) => {
 	}
 
 	const EnhancedBlockEdit = function( props ) {
-		const { attributes: { ampLayout }, name } = props;
+		const { isSelected, attributes: { ampLayout }, name } = props;
 
 		let inspectorControls;
 
 		if ( 'core/gallery' === name ) {
 			inspectorControls = setUpGalleryInspectorControls( props );
 		} else if ( 'core/image' === name ) {
 			inspectorControls = setUpImageInspectorControls( props );
-		} else if ( MEDIA_BLOCKS.includes( name ) || 0 === name.indexOf( 'core-embed/' ) ) {
+		} else if ( 'core/video' === name ) {
+			inspectorControls = isSelected ? VideoInspectorControls( props ) : null;
+		} else if ( 0 === name.indexOf( 'core-embed/' ) ) {
```

Comment:
Did this lose a condition?

```suggestion
		} else if ( MEDIA_BLOCKS.includes( name ) || 0 === name.indexOf( 'core-embed/' ) ) {
```

I realize that we've deprecated `AmpLayoutControl` and `AmpNoloadingToggle`, and that often the settings panel can show up empty per https://github.com/ampproject/amp-wp/issues/6508, but we haven't yet removed it entirely (which we should do in 2.2). So it seems `setUpInspectorControls` should still be applied to the other `MEDIA_BLOCKS`, namely here `core/image`.

Reply:
Yea that shouldn't be removed, good catch.